feat!(metrics): move extra processors after sumologic_schema (#3306)
SumoLogic/sumologic-kubernetes-collection: docs/v4-migration-doc.md

  Some Kubernetes objects, for example statefulsets, have a tight (63 characters) limit for their names. Because of that, we truncate the
  prefix that is attached to the names. In particular, the value under key `fullnameOverride` will be truncated to 22 characters.

- Moving extra processors in metrics pipeline after sumologic_schema processor

>   This has been changed in order to make this coherent with pipeline for logs.

Reviewer: ```suggestion
  This has been changed in order to make the behaviour consistent with the logs pipeline.
```
